python/tvm/driver/tvmc/compiler.py

```diff
@@ -470,9 +470,19 @@ def compile_model(
                 lib = graph_module.lib if use_vm else graph_module.get_lib()
                 # TODO lib.get_source call have inconsistent behavior for unsupported
                 #      formats (@leandron).
-                dumps[source_type] = lib.get_source(source_type)
+                try:
+                    dumps[source_type] = lib.get_source(source_type)
+                except tvm.TVMError:
+                    pass
```

Comment:
What do you think, maybe we need to dump also `TVMError` in case if we were not able to get source?